canvas_account_admin_tools/settings/local.py

```diff
@@ -64,6 +64,9 @@
          },
       }
    },
+   'cross_listing': {
+      'relative_url': 'accounts/8/external_tools/99'
```

Comment:
Seems like we could use the `account_admin` `relative_url` for all the tests under the dashboard...? They all start there, right?